docs/application/environment.md

```diff
@@ -126,8 +129,28 @@ Stop the environment, the system will stop the resources in the environment in s
 
 ![app-env-stop](/img/v0.5.0/application/environment/env-stop.png)
 
-
 ## Delete Environment
 
 1. Click `Application Management` -> `Environment` in the navigation bar to enter the environment list.
 2. Ensure no resources depend on this environment, select the environment you need to delete and click `Delete` to complete the deletion.
+
+## Set Default Environment
+
+The default environment accessed after logging in for the first time in Walrus is the `local` environment under the `default` project. You can also customize the default environment:
+
+1. In the breadcrumb `Environments` drop-down option at the top, click ![star-line](/img/v0.5.0/application/environment/star-line.png) in front of the environment to set it as the default access environment.
+
+   ![set-default-env-en](/img/v0.5.0/application/environment/set-default-env-en.png)
+
+2. If the project corresponding to the environment is deleted, this setting will also be deleted.
+3. When there is a default environment, logging in or clicking the menu `Applications` will enter the default environment details page.
+
+## Unset Default Environment
+
+In the breadcrumbs `Environments` drop-down option at the top, click ![star-fill](/img/v0.5.0/application/environment/star-fill.png) in front of the environment to cancel the default environment.
+
+![unset-default-env-en](/img/v0.5.0/application/environment/unset-default-env-en.png)
+
+1. After unset or deleting the default environment, the default access after successful login is the first environment under the first project.
+2. After unset the default environment, click the menu `Applications`, and the page entered by default is the recently visited page (project list, project details, environment details).
+3. After deleting the default environment, click the menu `Applications` and the default access is the first environment under the project.
```

Comment:
After unsetting or deleting the default environment, the default access after a successful login will be the first environment under the first project.

After unsetting the default environment, clicking the Applications menu will take you to the recently visited page (project list, project details, environment details).

After deleting the default environment, clicking the Applications menu will grant default access to the first environment under the project.

revised by GPT.